pyrule_compendium/__init__.py

```diff
@@ -88,7 +94,12 @@ def get_all(self, timeout: types.timeout=None) -> Union[dict, list]:
         if not timeout:
             timeout = self.default_timeout
 
-        return api_req(self.api.base_url, timeout)
+        if self.master_mode:
+            res: dict = api_req(self.api.base_url, timeout)
+            res["monsters"] += api_req(self.master_api.base_url, timeout)
```

Comment:
Are two requests necessary? I would rather we just have one.

Reply:
Without a second request, there isn’t a way to get data from `master_mode` since it’s a separate endpoint. If the user wants only one request, they can specify it when initializing the class.